pkg/reconciler/tenancy/bootstrap/bootstrap_controller.go

```diff
@@ -85,12 +89,12 @@ func NewController(
 // state and bootstrap resources from the configs/<lower-case-type> package.
 type controller struct {
 	controllerName string
+	config         *rest.Config
+	queue          workqueue.RateLimitingInterface
 
-	queue workqueue.RateLimitingInterface
-
-	dynamicClient dynamic.ClusterInterface
-	crdClient     apiextensionclientset.ClusterInterface
-	kcpClient     kcpclient.ClusterInterface
+	dynamicClient    dynamic.Interface
+	crdClient        apiextensionclientset.Interface
```

Comment:
crdClusterClient

Reply:
@sttts Modified this to be `crdClusterClient` and `dynamicClient`, but thinking about it - this could point towards the base cluster or cluster depending on the context which is passed. Do we follow the `*ClusterClient` convention hereon for all controllers, or shall we keep this as is.